SEMANTIC_CONVENTIONS.md

```diff
@@ -28,33 +28,106 @@ We use certain data types for the properties we expect.
+|----------------------|------------------------------------------------------------------------------------------------------|--------------------------------------|-------------|
+| name                 | The name of the event                                                                                | text                                 | MANDATORY   |
+| startDate            | Time of start for the event                                                                          | date                                 | MANDATORY   |
+| endDate              | Time of end for the event                                                                            | date                                 | OPTIONAL    |
+| url                  | A link to the specific event page for this event. Same as source.event.page.                         | url                                  | RECOMMENDED |
+| type                 | The type of event, for example `concert`, `????` more examples please                                | text ??? maybe enum would be better? | RECOMMENDED |           
+| category             | The category of an event, for example `MUSIC`, `ART` or `TECH`, see EventCategory enum               | enum\<EventCategory>                 | RECOMMENDED |
+| description          | Text describing this event                                                                           | text                                 | RECOMMENDED |
+| description.markdown | Text describing this event, but formatted with markdown.                                             | text                                 | OPTIONAL    |
+| recurrence.type      | Describing how often an event happens. Once, rarely or very often.                                   | enum\<RecurrenceType>                | RECOMMENDED |
+| recurrence.interval  | Describing the interval with which the event recurs                                                  | text                                 | OPTIONAL    |
+| tags                 | A list of tags. TODO how to describe?                                                                | list\<text>                          | RECOMMENDED |
+| registration         | If this is a free event, a event which requires registration or a event which requires a paid ticket | enum\<registration>                  | OPTIONAL    |
+| pictureUrl           | Url to a picture to be shown                                                                         | url                                  | DEPRECATED  |
+| pictures.json        | A json array of pictures for the event. The first picture should be the featured one.                | json\<image>                         | OPTIONAL    |
+| collectorName        | Name of the collector which collected this event                                                     | text                                 | MANDATORY   |
```

Comment:
not sure if we should make this mandatory, currently it is not

Reply:
ok. I just documented the current status. I thought it was mandatory, but then I will change it.